nlp_primitives/mean_characters_per_word.py

```diff
@@ -29,17 +46,8 @@ class MeanCharactersPerWord(TransformPrimitive):
     default_value = 0
 
     def get_function(self):
-        def mean_characters_per_word(x):
-            x = x.reset_index(drop=True).fillna('')
-            # replace end-of-sentence punctuation with space
-            p = re.escape('!,.:;?')
-            end_of_sentence_punct = re.compile('[%s]+$|[%s]+ |[%s]+\n' % (p, p, p))
-            x = x.str.replace(end_of_sentence_punct, ' ')
-            # build DF of split words, and calculate avg length
-            df = x.str.split(expand=True)
-            df = df.reset_index()
-            df = df.melt(id_vars='index', var_name='word_index', value_name='word')
-            df['n_characters'] = df['word'].str.len()
-            results = df.groupby(by=['index']).mean(numeric_only=False)['n_characters']
-            return results
+        def mean_characters_per_word(series):
+            series = series.reset_index(drop=True)
```

Comment:
Do we need to do this `reset_index` call?

Reply:
I don't think so. It was already there, and I wasn't sure if it was capturing something I'm not familiar with